x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plugin/TextFormat.java

```diff
@@ -197,6 +204,7 @@ String maybeEscape(String value, Character delimiter) {
         boolean hasHeader(RestRequest request) {
             String header = request.param(URL_PARAM_HEADER);
             if (header == null) {
+                //TODO PG in most places we only assume one accept header
```

Comment:
Good to know. Having multiple header fields with same name is legal though, I guess we should leave it as is.